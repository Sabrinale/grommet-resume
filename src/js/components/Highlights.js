import React from 'react'

import { Box } from 'grommet'

export const Highlights = ({ children }) => (
  <Box
    align="start"
    basis="auto"
    pad={{ left: '0px', right: '0px', top: '0px', bottom: '0px' }}
    margin={{ bottom: 'large', top: 'xsmall' }}
  >
    {children}
  </Box>
)
