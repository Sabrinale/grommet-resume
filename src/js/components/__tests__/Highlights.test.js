import React from 'react'
import renderer from 'react-test-renderer'

import { Highlights } from '../Highlights'

test('renders description', () => {
  const component = renderer.create(<Highlights>test</Highlights>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
