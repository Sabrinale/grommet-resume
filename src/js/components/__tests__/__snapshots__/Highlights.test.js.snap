// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders description 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 kFdcfG"
>
  test
</div>
`;
